src/components/settings/SettingsLMSTab/LMSConfigs/BlackboardConfig.jsx

```diff
@@ -134,7 +146,7 @@ const BlackboardConfig = ({
       if (!appKey) {
         try {
           const response = await LmsApiService.fetchBlackboardGlobalConfig();
-          appKey = response.data.results[0].app_key;
+          appKey = response.data.results[response.data.results.length - 1].app_key;
```

Comment:
youch, caught this when testing. The backend is returning the list from oldest -> newest, so we need to fetch the last in the list not the first.